src/app/components/Button/index.tsx

```diff
@@ -29,7 +31,10 @@ export default function Button({
       className={classNames(
         direction === "row" ? "flex-row" : "flex-col",
         fullWidth && "w-full",
-        fullWidth ? "px-0 py-2" : "px-7 py-2",
+        fullWidth ? "px-0 py-2" : "",
+        halfWidth && "w-1/2",
+        halfWidth ? "px-0 py-2 mx-2" : "",
+        !(halfWidth || fullWidth) && "px-7 py-2",
```

Comment:
hmmm, I think this is a bit hard to read
what means fullWidth vs. halfWidth
and why can both be true?

Reply:
I didn't understand the question, here we check if neither of the two is set and apply the original thing just so as not to break things.

Earlier (before this PR):
`fullWidth ? "px-0 py-2" : "px-7 py-2"`

Full Width Button
![181014351-cab1358f-9f30-47e1-97af-39ec7547cc38](https://user-images.githubusercontent.com/64399555/182786565-b1300b68-12ad-45fe-9709-e4a5d9d966a0.png)

Half Width Button
![Screenshot from 2022-08-04 12-42-47](https://user-images.githubusercontent.com/64399555/182786575-fbf27d82-3965-4c4b-acf2-d5100f704db6.png)